assets/js/components/Dashboard/PaMessageForm/SelectStationsAndZones/SelectZonesPage.tsx

```diff
@@ -42,13 +42,17 @@ const SelectZonesPage = ({
   places,
 }: Props) => {
   const routeToRouteIDMap = useRouteToRouteIDsMap();
+  const getInitialPlacesWithSelectedSigns = () =>
+    places.filter((p) => p.screens.some((s) => value.includes(s.id)));
 
-  const selectedRoutes = useMemo(() => {
+  // eslint-disable-next-line react/hook-use-state
+  const [initialPlacesWithSelectedSigns] = useState<Place[]>(
+    getInitialPlacesWithSelectedSigns,
+  );
```

Comment:
I'm sorry for the churn on this. I'm asking for `getInitialPlacesWithSelectedSigns` to be inlined into the call to `setState` (or `useRef` if you end up going that route) because it:

- is not used elsewhere
- is misleadingly named because it actually gets places with selected signs _at any point_ but you are currently only using to get the initial places with selected signs

```typescript
  // eslint-disable-next-line react/hook-use-state
  const [initialPlacesWithSelectedSigns] = useState<Place[]>(() =>
    places.filter((p) => p.screens.some((s) => value.includes(s.id))),
  );
```

Reply:
Agree with this — that's exactly why I inlined the function in my proposed change.

Reply:
O I see. I misunderstood.